Modelica/Mechanics/Translational.mo

```diff
@@ -3280,6 +3399,384 @@ following references, especially (Armstrong and Canudas de Wit 1996):
 </html>"));
     end IdealRollingWheel;
 
+    model RollingResistance "Resistance of a rolling wheel"
+      extends Modelica.Mechanics.Translational.Interfaces.PartialForce;
+      import Modelica.Constants.pi;
+      parameter SI.Force fNormal(start=0) "Wheel load due to gravity";
```

Comment:
The name `fNormal` may not be a good choice as the normal force is calculated by the expression

`fNormal*cos(atan(inclination_internal))`

in line 3440. As I understand the model `fNormal` is the weight (mass * acceleration of gravity) = `m * g` only, see line 3577.